### PR TITLE
Allow using fsaverage as a template MRI, e.g. for EEG studies without MR scans

### PR DESCRIPTION
**Pull request by agramfort**
cc @apmellot @dengemann

### CODE REVIEW

**Review by hoechenberger — CHANGES_REQUESTED** [review]
Looks quite good already! Nice!
Unfortunately CI is broken b/c of OpenNeuro problems.
@agramfort If the `ds000248_no_mri` test passes locally for you, we may be able to move ahead anyway and merge. But before that, there are a few things missing:

- [ ] changelog entry
- [x] entry in `datasets.py`
- [x] `run_tests.py` needs to be updated
- [x] Circle config needs to be updated
- [x] The new config option should be added to `docs/source/settings/source/bem.md`

**Comment by hoechenberger** [comment]
@agramfort I think I'd like to rename the param from `use_mri_template` to `use_template_mri`, WDYT?

File: tests/configs/config_ds000248_no_mri.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""
+MNE Sample Data
+"""
+
+study_name = 'ds000248'
+bids_root = '~/mne_data/ds000248'
+deriv_root = '~/mne_data/ds000248/derivatives/mne-bids-pipeline-no-mri'
```
Comment (agramfort) [inline tests/configs/config_ds000248_no_mri.py:7]: should I remove this line now @hoechenberger?

File: tests/datasets.py [hunk 2]
```diff
@@ -109,6 +109,14 @@ class DATASET_OPTIONS_T(TypedDict):
             'derivatives/freesurfer/subjects/sub-01/mri/aparc.a2009s+aseg.mgz'
         ],
     },
+    'ds000248_no_mri': {
+        'git': '',
+        'openneuro': 'ds000248',
+        'osf': '',
+        'web': '',
+        'include': ['derivatives/freesurfer/subjects/sub-01'],
+        'exclude': ['derivatives/freesurfer'],
```
Comment (agramfort) [inline tests/datasets.py:118]: can you also exclude the "anat" folder?

**Review by agramfort — COMMENTED** [review]
for @hoechenberger

**Comment by agramfort** [comment]
fine with use_template_mri

>

File: tests/datasets.py (hunk 2 above)
Comment (hoechenberger) [reply tests/datasets.py:118]: The current combination of include and exclude that I've added here actually doesn't make sense … We only need `derivatives/freesurfer`, right?? Will update

File: tests/datasets.py [hunk 3]
```diff
@@ -114,8 +114,8 @@ class DATASET_OPTIONS_T(TypedDict):
         'openneuro': 'ds000248',
         'osf': '',
         'web': '',
-        'include': ['derivatives/freesurfer/subjects/sub-01'],
-        'exclude': ['derivatives/freesurfer'],
+        'include': ['derivatives/freesurfer'],
```
Comment (agramfort) [inline tests/datasets.py:117]: We don’t need any freesurfer. Fsaverage should be fetched automatically by the pipeline

Comment (hoechenberger) [reply tests/datasets.py:117]: Oh, so … what do we even need then? 😅 just the MEG?

Comment (hoechenberger) [reply tests/datasets.py]: Changed it to downloading only the MEG for sub-01 now, and omitting the anat

**Comment by hoechenberger** [comment]
@agramfort It appears the fsaverage download doesn't work as intended (or not at all?), hence the test fails. I can look into this later today, unless you immediately have an idea as to where things might be going wrong :)

**Comment by agramfort** [comment]
I am on it

**Comment by agramfort** [comment]
should be fixed @hoechenberger

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
Thanks @agramfort, let's see if the artifact production works now as expected, and then proceed to merging.

**Comment by hoechenberger** [comment]
It doesn't show up in the examples gallery, but I'm going ahead and will be merging now, and will fix the gallery later (I think it's really just the TOC entry that is missing!)

**Comment by hoechenberger** [comment]
Ok I think I resolved it. Let's wait for CI to go green

**Comment by hoechenberger** [comment]
🥳 Thanks @agramfort!

**Pull request merged**